src/Umbraco.Web/BatchedDatabaseServerMessenger.cs

```diff
@@ -111,10 +113,15 @@ protected ICollection<RefreshInstructionEnvelope> GetBatch(bool create)
                 // the case if the asp.net synchronization context has kicked in
                 ?? (HttpContext.Current == null ? null : new HttpContextWrapper(HttpContext.Current));
 
-            // if no context was found, return null - we cannot not batch
+            // if no context was found, return null - we cannot batch
             if (httpContext == null) return null;
 
-            var key = typeof (BatchedDatabaseServerMessenger).Name;
+            return GetBatch(create, httpContext);
+        }
+
+        protected ICollection<RefreshInstructionEnvelope> GetBatch(bool create, HttpContextBase httpContext)
```

Comment:
Should we keep this to avoid a breaking change?
```cs
protected ICollection<RefreshInstructionEnvelope> GetBatch(bool create) {
    return GetBatch(create, null);
}
```

Reply:
I think so yes, but it's still there - hard to see on the segment shown here but if you expand the number of lines shown or view the full file I think you'll see that I've kept all the old signatures.

Reply:
Sorry..  All good then